<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>医院挂号系统登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_username">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_password">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>密  码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lE_username">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{border-radius: 10px;
border: 2px groove blue;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_passwd">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{border-radius: 10px;
border: 2px groove blue;
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="Bn_cancel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-radius: 10px;
background-color: rgb(251, 255, 169);}


QPushButton:hover{background-color: rgb(24, 160, 223);}</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Bn_login">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-radius: 10px;
background-color: rgb(251, 255, 169);}


QPushButton:hover{
background-color: rgb(24, 160, 223);
}

</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bn_network">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-radius: 10px;
background-color: rgb(251, 255, 169);}


QPushButton:hover{background-color: rgb(24, 160, 223);}</string>
   </property>
   <property name="text">
    <string>网络设置&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/背景图.jpg);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>label_username</zorder>
  <zorder>label_password</zorder>
  <zorder>lE_username</zorder>
  <zorder>LE_passwd</zorder>
  <zorder>Bn_cancel</zorder>
  <zorder>Bn_login</zorder>
  <zorder>Bn_network</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
